daemon/daemon.go

```diff
@@ -77,6 +77,10 @@ func (d *Daemon) ListServices(namespace string) ([]flux.ServiceStatus, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "checking service policies")
 	}
+	allServices, err := d.Manifests.ServicesWithPolicy(d.Checkout.ManifestDir(), policy.None)
```

Comment:
This seems like a round-about way of getting the result you want. To some extent it's an accident of history (automated and locked started as properties from somewhere else, blah etc.).

But now you always want to get all the services, with their policy, from the manifests. Why not do just it in one call? Then you don't have to have the special case of `policy.None`.